platform/ios/src/MLNMapView.mm

```diff
@@ -6428,7 +6445,7 @@ - (void)locationManager:(__unused id<MLNLocationManager>)manager didUpdateHeadin
         if (headingDirection >= 0 && self.userTrackingMode == MLNUserTrackingModeFollowWithHeading
             && self.userTrackingState != MLNUserTrackingStateBegan)
         {
-            [self _setDirection:headingDirection animated:YES];
+            [self _setDirection:headingDirection shouldCenter:YES animated:YES];
```

Comment:
I understand `_setDirection` needs the current location so that it can do the animation in one go, but maybe consider passing the center instead of a boolean. I don't think the `_setDirection` method should have anything to do with the location manager.

Reply:
I changed parameter from `shouldCenter` to `center`. And so public `setDirection` method passes `kCLLocationCoordinate2DInvalid` to private `_setDirection` to avoid centering.

Reply:
@louwers is there anything else that needs to be changed or you are fine with it? 

This is a quite important feature for us, and we want to have it merged (and released) as soon as possible.

Reply:
Looks good now, thanks!